wct.conf.json

```diff
@@ -13,8 +13,7 @@
       },
       {
         "browserName": "safari",
-        "platform": "OS X 10.12",
-        "version": "10"
+        "platform": "macOS 10.13"
```

Comment:
wow, really, macOS not OS X? I'd not guess that

Reply:
Me too, but this saves me everytime https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/